go/tunnels/manager_test.go

```diff
@@ -16,19 +16,20 @@ import (
 )
 
 var (
-	serviceUrl           = ServiceProperties.ServiceURI
+	serviceUrl           = "https://localhost:9901/"
```

Comment:
Was this changed for testing?

Reply:
Oh, yes that change was not supposed to be in the PR.

Reply:
Reverted.